main.go

```diff
@@ -193,6 +198,15 @@ func findFiles(patterns []string) (_ []string, err error) {
 	return sortedFiles, err
 }
 
+func modify(filename string, content []byte) error {
```

Comment:
I understand the intent behind calling this modify (as in modify vs preview), and that does help readability down below, but in isolation, this function "modify" has less meaning.

Consider just inlining it instead of declaring the function, since it's a single function call.